### PR TITLE
Full 9.1.13.0

### PR DESCRIPTION
**Pull request by nadavshatz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,30 +1,78 @@
...
 
-RUN apt-get update
-RUN apt-get install -y wget
-RUN apt-get install -y libc6-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+RUN add-apt-repository -y ppa:webupd8team/java
 
-# jruby
-ENV JRUBY_VERSION 9.1.8.0
-ENV JRUBY_SHA256 20ac501c99a7cb3cf53ded64ac1b8bb6e0b0f6ba34a41b8bacc9715cd4bb2601
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
```
Comment (gmaliar) [inline Dockerfile]: should be changed from `trust` to `xenial` no?
alternatively you can change `trusty` to `$(lsb_release -cs)` and it'll be future proof.

File: Dockerfile [hunk 2]
```diff
@@ -1,30 +1,78 @@
...
-RUN gem install bundler \
-	&& bundle config --global path "$GEM_HOME" \
-	&& bundle config --global bin "$GEM_HOME/bin"
+ENV BUNDLE_PATH="$GEM_HOME" \
+    BUNDLE_BIN="$GEM_HOME/bin" \
+    BUNDLE_SILENCE_ROOT_WARNING=1 \
+    BUNDLE_APP_CONFIG="$GEM_HOME"
+ENV PATH $BUNDLE_BIN:$PATH
+RUN mkdir -p "$GEM_HOME" "$BUNDLE_BIN" \
+    && chmod 777 "$GEM_HOME" "$BUNDLE_BIN"
```
Comment (gmaliar) [inline Dockerfile:46]: quite aggressive setting this as 777 isn't it?

**Review by gmaliar — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (nadavshatz) [reply Dockerfile:46]: Does it matter running inside the container? almost everything runs as root anyway no?

File: Dockerfile (hunk 1 above)
Comment (nadavshatz) [reply Dockerfile]: 👍

**Comment by nadavshatz** [comment]
@gmaliar updated.

**Review by gmaliar — APPROVED** [review]

**Pull request merged**